.gitattributes

```diff
@@ -9,3 +9,18 @@ Assets/Music/510953__theojt__cinematic-battle-song.mp3 filter=lfs diff=lfs merge
 Assets/Music/510953__theojt__cinematic-battle-song.mp3.meta filter=lfs diff=lfs merge=lfs -text
 Assets/Music/A1-Cave.mp3 filter=lfs diff=lfs merge=lfs -text
 Assets/Music/A1-Cave.mp3.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/ filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music/510953__theojt__cinematic-battle-song.mp3 filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music/510953__theojt__cinematic-battle-song.mp3.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music/A1-Cave.mp3 filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Music/A1-Cave.mp3.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Ambient filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Ambient.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Foleys filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Foleys.meta filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Mixer filter=lfs diff=lfs merge=lfs -text
+Assets/Resources/Sounds/Mixer.meta filter=lfs diff=lfs merge=lfs -text
```

Comment:
can you use wildcards ( ** ) here?

Reply:
I don't know, I used the official LFS tutorial using the Terminal and never touched the file manually.

Reply:
It works exactly like .gitignore
You can track all files in a folder with 
`{path}/** filter=lfs diff=lfs merge=lfs -text`
Or all files of a certain type with 
`{path}/*.[fileending] filter=lfs diff=lfs merge=lfs -text`
Would probably be a better way than adding them all one by one

Reply:
You can also use 
`git lfs track {path}/**`

Reply:
will do!